static/scss/base/_typography.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * TYPOGRAPHIE
+ */
+
+// ROBOTO MONO
+@import url(https://fonts.googleapis.com/css?family=Roboto+Mono:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic);
+
+// PLAYFAIR DISPLAY
+@import url(https://fonts.googleapis.com/css?family=Playfair+Display:400,400italic,700,700italic,900,900italic);
```

Comment:
Sure, done in https://github.com/open-craft/opencraft/pull/93/commits/1a509be1e7ce747ecff2ccbe5d4b0f16c6fd955b